src/Servers/IIS/AspNetCoreModuleV2/CommonLib/config_utility.h

```diff
@@ -14,6 +14,7 @@ class ConfigUtility
     #define CS_ASPNETCORE_HANDLER_SETTINGS                   L"handlerSettings"
     #define CS_ASPNETCORE_HANDLER_VERSION                    L"handlerVersion"
     #define CS_ASPNETCORE_DEBUG_FILE                         L"debugFile"
+    #define CS_ASPNETCORE_ENABLE_OUT_OF_PROCESS_CONSOLE_REDIRECTION L"EnableOutOfProcessConsoleRedirection"
```

Comment:
I guess I'm a few hours late to this, but I prefer DisableOutOfProcessConsoleRedirection because it defaults to false.

Reply:
Yeah there is one that default to true already. I'm not opinionated but the guidance is to have the positive be "true".

Reply:
I'm sold on both arguments (that this should be like AppContext switches and thus `false` by default, and that if there are other ANCM switches that default to `true` it's fine for this one to do so), so 🤷‍♂.

I **do** thing we should match casing with the others though. So it should be:

```suggestion
    #define CS_ASPNETCORE_ENABLE_OUT_OF_PROCESS_CONSOLE_REDIRECTION L"enableOutOfProcessConsoleRedirection"
```